tests/test_aten_functions.py

```diff
@@ -1039,6 +1039,170 @@ def fn(x):
     check_functions_are_equivalent(fn, device, [x])
 
 
+@pytest.mark.parametrize("dtype", [torch.float32, torch.float64, torch.bfloat16])
+def test_aten_cosh_basic(device: str, dtype: torch.dtype):
+    """Test aten.cosh basic functionality with floating point numbers"""
+    # Skip float16 on CPU as MAX doesn't support f16 on CPU
+    if device == "cpu" and dtype == torch.float16:
+        pytest.skip("float16 not supported on CPU in MAX")
+
+    def fn(x):
+        return aten.cosh(x)
+
+    # Test with positive, negative, and zero values
+    # cosh(0) = 1, cosh is even function: cosh(-x) = cosh(x)
+    x = torch.tensor([-2.0, -1.0, -0.5, 0.0, 0.5, 1.0, 2.0], dtype=dtype, device=device)
+    check_functions_are_equivalent(fn, device, [x])
+
+
+def test_aten_cosh_2d_tensor(device: str):
+    """Test aten.cosh with 2D tensor"""
+
+    def fn(x):
+        return aten.cosh(x)
+
+    x = torch.tensor(
+        [[-1.5, -0.5], [0.0, 1.0], [1.5, 2.5]], dtype=torch.float32, device=device
+    )
+    check_functions_are_equivalent(fn, device, [x])
+
+
+def test_aten_cosh_3d_tensor(device: str):
+    """Test aten.cosh with 3D tensor"""
+
+    def fn(x):
+        return aten.cosh(x)
+
+    x = torch.randn(2, 3, 4, dtype=torch.float32, device=device)
+    check_functions_are_equivalent(fn, device, [x])
+
+
+def test_aten_cosh_large_values(device: str):
+    """test aten.cosh with large values (may approach infinity)"""
+
+    def fn(x):
+        return aten.cosh(x)
+
+    # large values will produce large results due to exponential growth
+    x = torch.tensor([-5.0, -3.0, 3.0, 5.0], dtype=torch.float32, device=device)
+    check_functions_are_equivalent(fn, device, [x])
+
+
+def test_aten_cosh_small_values(device: str):
+    """test aten.cosh with small values near zero"""
+
+    def fn(x):
+        return aten.cosh(x)
+
+    # for small x, cosh(x) ≈ 1 + x²/2
+    x = torch.tensor(
+        [-0.1, -0.01, -0.001, 0.001, 0.01, 0.1], dtype=torch.float32, device=device
+    )
+    check_functions_are_equivalent(fn, device, [x])
+
+
+def test_aten_cosh_single_element(device: str):
+    """test aten.cosh with single element tensor"""
+
+    def fn(x):
+        return aten.cosh(x)
+
+    x = torch.tensor([1.5], dtype=torch.float32, device=device)
+    check_functions_are_equivalent(fn, device, [x])
+
+
+def test_aten_cosh_scalar_tensor(device: str):
+    """Test aten.cosh with scalar tensor"""
+
+    def fn(x):
+        return aten.cosh(x)
+
+    x = torch.tensor(1.0, dtype=torch.float32, device=device)
+    check_functions_are_equivalent(fn, device, [x])
+
+
+@pytest.mark.parametrize("dtype", [torch.float32, torch.float64, torch.bfloat16])
+def test_aten_sinh_basic(device: str, dtype: torch.dtype):
+    """Test aten.sinh basic functionality with floating point numbers"""
+    # Skip float16 on CPU as MAX doesn't support f16 on CPU
+    if device == "cpu" and dtype == torch.float16:
+        pytest.skip("float16 not supported on CPU in MAX")
```

Comment:
```suggestion
        pytest.xfail("float16 not supported on CPU in MAX")
```
Let's use an xfail instead since it should work. I serves as a reminder that we should remove the xfail at some point.